db/db_test.go

```diff
@@ -2,45 +2,68 @@ package db
 
 import (
 	"fmt"
-	"testing"
 	"github.com/yoiner-castillo-globant/GBcamp/db"
+	"testing"
 )
 
 func TestCreate(t *testing.T) {
-	db.Create("test1", "50")
-	length := len(db.Datos)
-	if length == 1 {
-		t.Errorf("Create was incorrect, got: %d, want: %d.", length, 1)
+
+	data := db.NewMemoryDB()
+
+	err := data.Create("test1", "50")
```

Comment:
be aware, the current `Create` method returns a `string` not `error` maybe need to be updated